File: sdkwork-app-sdk-csharp/Models/PositionDetailVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class PositionDetailVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Name { get; set; }
        public string? Code { get; set; }
        public string? Description { get; set; }
        public int? OwnerId { get; set; }
        public int? ParentId { get; set; }
        public int? Level { get; set; }
        public bool? IsActive { get; set; }
        public int? SortOrder { get; set; }
        public TreeNodePath? TreePath { get; set; }
        public string? FullPathName { get; set; }
        public string? Category { get; set; }
        public string? LevelName { get; set; }
        public int? RequiredExperienceYears { get; set; }
        public string? RequiredEducation { get; set; }
        public int? MaxMemberCount { get; set; }
        public int? CurrentMemberCount { get; set; }
        public bool? IsFull { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/position_detail_vo.go
package types

// 岗位详情响应
type PositionDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Name string `json:"name"`
	Code string `json:"code"`
	Description string `json:"description"`
	OwnerId int `json:"ownerId"`
	ParentId int `json:"parentId"`
	Level int `json:"level"`
	IsActive bool `json:"isActive"`
	SortOrder int `json:"sortOrder"`
	TreePath TreeNodePath `json:"treePath"`
	FullPathName string `json:"fullPathName"`
	Category string `json:"category"`
	LevelName string `json:"levelName"`
	RequiredExperienceYears int `json:"requiredExperienceYears"`
	RequiredEducation string `json:"requiredEducation"`
	MaxMemberCount int `json:"maxMemberCount"`
	CurrentMemberCount int `json:"currentMemberCount"`
	IsFull bool `json:"isFull"`
}
